packages/excalidraw/actions/actionFlip.ts

```diff
@@ -67,7 +68,7 @@ export const actionFlipVertical = register({
 
 const flipSelectedElements = (
   elements: readonly ExcalidrawElement[],
-  elementsMap: NonDeletedElementsMap,
+  elementsMap: NonDeletedElementsMap | NonDeletedSceneElementsMap,
```

Comment:
annoying but necessary for now. We can later remove this by removing the brand on `NonDeletedElementsMap` once we make `NonDeleted<>` type into a branded type of its own.